zitadel/v2/instance/const.go

```diff
@@ -0,0 +1,18 @@
+package instance
+
+const (
+	instanceNameVar                = "instance_name"
+	firstOrgNameVar                = "first_org_name"
+	firstOrgIDVar                  = "first_org_id"
+	customDomainVar                = "custom_domain"
+	ownerUserNameVar               = "owner_user_name"
+	ownerUserIDVar                 = "owner_user_id"
+	ownerEmailVar                  = "owner_email"
+	ownerIsEmailVerifiedVar        = "owner_is_email_verified"
+	ownerFirstNameVar              = "owner_first_name"
+	ownerLastNameVar               = "owner_last_name"
+	ownerPreferredLanguageVar      = "owner_preferred_language"
+	ownerPasswordVar               = "owner_password"
+	ownerPasswordChangeRequiredVar = "owner_password_change_required"
+	defaultLanguageVar             = "default_language"
+)
```

Comment:
the add instance in the system proto is not designed very well... would it make sense to struct it in terraform a bit more? will have to change the proto anyway soon (in regard of service accounts in newly created instances)